templates/advanced/package.yaml

```diff
@@ -4,15 +4,18 @@ name: Advanced Template Package
 # Required: A description of what's in the action package.
 description: Action package description
 
-# Required: A link to where the documentation on the package lives.
-documentation: https://github.com/...
+# Required: Point to main documentation markdown file for the package.
+documentation: README.md
```

Comment:
Have we decided on picking configuration over convention? Like, what would be an use-case where the `documentation:` field won't be a _README.md_? Normally, any action pack contains the main _README.md_, which can also point to additional documentation found in the _docs_ dir if needed.

The _package.png_ idiom works great so far, so what would be an advantage here on bringing them in the _package.yaml_?

Reply:
Good point, we agreed to convention on this.